### PR TITLE
Create website and add default PaymentGateway on signup (GWY-3690)

### PR DESCRIPTION
**Pull request by dkotin**
Resolves: GWY-3690

### CODE REVIEW

File: spec/definitions/Signup.yaml [hunk 1]
```diff
@@ -28,3 +29,12 @@ properties:
     description: The user password
     type: string
     format: password
+  url:
+    description: The user's website domain address
+    type: string
+  currencies:
```
Comment (XaosSintez) [inline spec/definitions/Signup.yaml]: You must define type of array elements

File: spec/definitions/Signup.yaml [hunk 2]
```diff
@@ -28,3 +29,12 @@ properties:
     description: The user password
     type: string
     format: password
+  url:
```
Comment (XaosSintez) [inline spec/definitions/Signup.yaml]: May be better not `url`, but something like `website` or `domain` ?

Comment (slavcodev) [inline spec/definitions/Signup.yaml]: +1 for `website`

File: spec/definitions/Signup.yaml [hunk 3]
```diff
@@ -28,3 +29,12 @@ properties:
...
+    description: The user's website domain address
+    type: string
+  currencies:
+    description: An array of ISO 3-letter currencies codes. Default: ['USD']
+    type: array
+  merchantCategoryCode:
+    description: An array of merchant cateogries ids. Default: 5734 (Computer Software Stores)
+    type: array
```
Comment (slavcodev) [inline spec/definitions/Signup.yaml]: Also need a items spec

**Comment by XaosSintez** [comment]
:+1:

**Pull request merged**